pkg/routers/api_v1.go

```diff
@@ -110,9 +123,10 @@ func WithAPIV1Router(opt *APIV1Options, r chi.Router) {
 	}
 
 	r.Route("/v1", func(r chi.Router) {
-		r.With(getConfigTimer).Get("/config", opt.handlers.config)
-		r.With(activateTimer).Post("/activate", opt.handlers.activate)
-		r.With(trackTimer).Post("/track", opt.handlers.trackEvent)
-		r.With(overrideTimer).Post("/override", overrideHandler)
+		r.With(getConfigTimer, opt.oAuthMiddleware.AuthorizeAPI).Get("/config", opt.handlers.config)
+		r.With(activateTimer, opt.oAuthMiddleware.AuthorizeAPI).Post("/activate", opt.handlers.activate)
+		r.With(trackTimer, opt.oAuthMiddleware.AuthorizeAPI).Post("/track", opt.handlers.trackEvent)
+		r.With(overrideTimer, opt.oAuthMiddleware.AuthorizeAPI).Post("/override", overrideHandler)
+		r.With(createAccesstokenTimer).Post("/oauth/token", opt.oAuthHandler.CreateAPIAccessToken)
```

Comment:
Hm, I'm curious if this even makes sense under `/v1`? This API versioning is for the Agent API, the auth flow is more or less prescribed by the OAuth 2.0 spec and not under the same versioning scheme as our API. Thoughts?

Reply:
The SDK key header is required, and included in the claims. Does it make sense to say, in theory, this may need to be versioned alongside the Agent API, because we might remove or change the SDK key header?

Reply:
The SDK key changing seems very unlikely so I would agree with moving this out of `/v1`. Probably its path should include `api` somewhere to indicate that it gives API access, as opposed to admin access.